lambdas/data-migration2/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { getKnexClient } from '@cumulus/db';
+
+import { migrateExecutions } from './executions';
+
+export interface HandlerEvent {
+  env?: NodeJS.ProcessEnv
+}
+
+export const handler = async (event: HandlerEvent): Promise<string> => {
+  const env = event.env ?? process.env;
+
+  const knex = await getKnexClient({ env });
+
+  try {
+    const executionsMigrationSummary = await migrateExecutions(env, knex);
+
+    return `
+      Migration summary:
```

Comment:
As above, on error we lose this summary - should we consider defensive behavior here and  make sure of logging/dumping/? on an uncaught error?

Reply:
TODO: use Logger to log migration summary before return.

Reply:
Hmm so actually we do log the migration summary on completion and any time there's a migration error. I'm wondering if that's sufficient knowing that there is work planned to develop additional reporting on the status here (or at least discussing that possibility)?

Reply:
The relevant logging is here:

https://github.com/nasa/cumulus/pull/2029/files/63551946807ddb72565bcce1c46cbcfe7e7bff10#diff-0c998a701074d6b429e241827a83826f1590334993cafb70f1d57cf79b5b119dR86

The idea here is if there's a problem migrating a record we catch and log the error outside of the Lambda handler in `migrateExecutions`.